src/Amazon.AspNetCore.DataProtection.SSM/XmlDeletableKeyManager.cs

```diff
@@ -0,0 +1,85 @@
+﻿/*
+Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License").
+  You may not use this file except in compliance with the License.
+  A copy of the License is located at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  or in the "license" file accompanying this file. This file is distributed
+  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+  express or implied. See the License for the specific language governing
+  permissions and limitations under the License.
+ */
+
+using Microsoft.AspNetCore.DataProtection.Internal;
+using Microsoft.AspNetCore.DataProtection.KeyManagement;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace Amazon.AspNetCore.DataProtection.SSM
+{
+#if NET9_0_OR_GREATER
+    /// <summary>
+    /// Extension of XmlKeyManager which supports key deletion.
```

Comment:
nit: I would suggest adding a comment why we are doing this. Something like "XmlKeyManager implements IDeletableKeyManager but doesn't have the declaration for implementing IDeletableKeyManager. We created our wrapper to have the IDeletableKeyManager declaration."

If you have posted your question on the ASP .NET Core website then put the link here.

Reply:
Done. I assume you meant adding it as part of documentation comment, as `<para>`.